spec/frodo/concerns/api_spec.rb

```diff
@@ -211,6 +211,20 @@
         expect(subject).to be(true)
       end
     end
+
+    context 'with additional headers' do
+      let(:additional_header) { {"header" => '1' } }
+
+      before do
+        stub_request(verb, uri).to_return(body: body.to_json, headers: headers.merge!(additional_header))
```

Comment:
Josh and I talked about modifying this test to be more explicit about what's going on. Right now we're relying on the fact that the test framework would raise an error about an external request if this stub wasn't here.